### ISSUE TITLE
Teamtalk QT's translation app is poorly accessible with screen readers

### ISSUE THREAD

**Issue by djgraco** [BearWare/TeamTalk5#1085]
Hello,
Teamtalk QT's translation app is poorly available with screen readers. Is there any possibility to convert the LNG language pack from Classic to QT?
There are blind translators who have been actively translating new versions of teamtalk. Currently, however, due to the unavailability of a translator, they will not be able to develop translations.
Hope this problem will be resolved somehow.
Best Regards

**Comment by bear101** [comment]
Unfortunately we cannot fix this issue. I'm afraid the only way is to use a text editor.

**Comment by bear101** [comment]
A quick google search revealed these alternative: https://www.topbestalternatives.com/qt-linguist/
